magik-language-server/src/main/java/nl/ramsolutions/sw/magik/languageserver/MagikLanguageServer.java

```diff
@@ -146,15 +146,15 @@ public NotebookDocumentService getNotebookDocumentService() {
 
   @Override
   public void connect(final LanguageClient newLanguageClient) {
-    this.languageClient = newLanguageClient;
+    this.languageClient = (MagikLanguageClient) newLanguageClient;
   }
 
   /**
    * Get the {@link LanguageClient}.
    *
    * @return Language client.
    */
-  public LanguageClient getLanguageClient() {
+  public MagikLanguageClient getLanguageClient() {
```

Comment:
Why is this needed? It is not used anywhere.

Reply:
This was used while I was testing custom notifications sent from the server but I left it here since it could be useful in the future and would be less of a hassle to refactor later on. I can of course remove it though